### PR TITLE
Use breakoutroomitem component in participant editor

### LINKED ISSUE #8986: Use breakout rooms item in breakout rooms participants editor
At the moment the participants editor uses the appnavigationitem component

### PR DESCRIPTION
**Pull request by marcoambrosini**
fix #8986 

Screenshot 2023-03-07 at 13 11 13

### CODE REVIEW

**Comment by marcoambrosini** [comment]
pretty painful rebase ahead

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue [hunk 1]
```diff
@@ -31,7 +31,11 @@
 			<template v-for="breakoutRoom in breakoutRooms">
 				<BreakoutRoomItem :key="breakoutRoom.token"
 					:breakout-room="breakoutRoom"
-					:main-conversation="mainConversation" />
+					:main-conversation="mainConversation">
+					<template v-for="participant in $store.getters.participantsList(breakoutRoom.token)">
```
[the remark this answers is not in the document]
Comment (marcoambrosini) [reply src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:35]: breakoutroom in that case is available in template only

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomItem.vue [hunk 2]
```diff
@@ -148,21 +152,29 @@ export default {
 			return this.roomToken !== this.$store.getters.getToken()
 		},
 
-		roomParticipants() {
-			return this.$store.getters.participantsList(this.roomToken)
-		},
-
 		canFullModerate() {
+			return !this.isParticipantsEditor && (this.participantType === PARTICIPANT.TYPE.OWNER || this.participantType === PARTICIPANT.TYPE.MODERATOR)
 			return this.participantType === PARTICIPANT.TYPE.OWNER || this.participantType === PARTICIPANT.TYPE.MODERATOR
```
Comment (Antreesy) [inline src/components/RightSidebar/BreakoutRooms/BreakoutRoomItem.vue:157]: And now we don't need this line

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomItem.vue [hunk 3]
```diff
@@ -148,21 +152,29 @@ export default {
...
 		},
 
 		canModerate() {
+			if (this.isParticipantsEditor) {
```
Comment (Antreesy) [inline src/components/RightSidebar/BreakoutRooms/BreakoutRoomItem.vue:161]: And i mean to change each new `if (this.isParticipantEditor)` entry in this file. Sorry, if you didn't get it

**Review by Antreesy — APPROVED** [review]

**Pull request merged**